extension/chrome/src/js/background.js

```diff
@@ -251,9 +228,9 @@ function pingAllExistingTabs() {
 
 
 // Click on extension icon
-chrome.browserAction.onClicked.addListener(function(tab) {
+chrome.browserAction.onClicked.addListener(function() {
 	chrome.storage.local.get(['currentStashBaseUrl'], function(items) {
-		chrome.tabs.create({'url': items.currentStashBaseUrl + '/'}, function(tab) {});
+		chrome.tabs.create({'url': `${items.currentStashBaseUrl  }/`}, function() {});
```

Comment:
[question] why removing `tab` and is the empty function required argument any more?